src/Illuminate/Cookie/Guard.php

```diff
@@ -65,7 +65,7 @@ protected function decrypt(Request $request)
 			{
 				$request->cookies->set($key, $this->decryptCookie($c));
 			}
-			catch (DecryptException $e)
+			catch (\Throwable $e)
```

Comment:
This is too vague... DecryptException is fine

Reply:
have you read the my comment? it's not fine because users are able to crash framework by setting cookies in a specific way. Otherwise this can be solved by checking if value is string here:

`	protected function decryptArray(array $cookie)
	{
		$decrypted = array();
		foreach ($cookie as $key => $value)
		{
                        // $value could be array here, check if string
			$decrypted[$key] = $this->encrypter->decrypt($value);
		}
		return $decrypted;
	}`

Reply:
But what is the exception and message you actually got in your case?